+ source ../common/common_lib.sh
++ source /root/mugen-riscv/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n '' ']'
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start testing...'
+ message='Start testing...'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start testing...'
Fri Jan  6 13:26:57 2023 - INFO  - Start testing...
+ test_execution kdump.service
+ service=kdump.service
++ date '+%Y-%m-%d %T'
+ log_time='2023-01-06 13:26:57'
+ test_restart kdump.service
+ service=kdump.service
+ systemctl restart kdump.service
Failed to restart kdump.service: Unit kdump.service not found.
+ CHECK_RESULT 5 0 0 'kdump.service restart failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='kdump.service restart failed'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'kdump.service restart failed'
+ LOG_ERROR 'kdump.service restart failed'
+ message='kdump.service restart failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'kdump.service restart failed'
Fri Jan  6 13:26:58 2023 - ERROR - kdump.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 34'
+ message='../common/common_lib.sh line 34'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 34'
Fri Jan  6 13:26:58 2023 - ERROR - ../common/common_lib.sh line 34
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status kdump.service
+ grep 'Active: active'
Unit kdump.service could not be found.
+ CHECK_RESULT 1 0 0 'kdump.service restart failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='kdump.service restart failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'kdump.service restart failed'
+ LOG_ERROR 'kdump.service restart failed'
+ message='kdump.service restart failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'kdump.service restart failed'
Fri Jan  6 13:27:05 2023 - ERROR - kdump.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 37'
+ message='../common/common_lib.sh line 37'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 37'
Fri Jan  6 13:27:05 2023 - ERROR - ../common/common_lib.sh line 37
+ return 0
+ systemctl stop kdump.service
Failed to stop kdump.service: Unit kdump.service not loaded.
+ CHECK_RESULT 5 0 0 'kdump.service stop failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='kdump.service stop failed'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'kdump.service stop failed'
+ LOG_ERROR 'kdump.service stop failed'
+ message='kdump.service stop failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'kdump.service stop failed'
Fri Jan  6 13:27:06 2023 - ERROR - kdump.service stop failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 39'
+ message='../common/common_lib.sh line 39'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 39'
Fri Jan  6 13:27:07 2023 - ERROR - ../common/common_lib.sh line 39
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status kdump.service
+ grep 'Active: inactive'
Unit kdump.service could not be found.
+ CHECK_RESULT 1 0 0 'kdump.service stop failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='kdump.service stop failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'kdump.service stop failed'
+ LOG_ERROR 'kdump.service stop failed'
+ message='kdump.service stop failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'kdump.service stop failed'
Fri Jan  6 13:27:14 2023 - ERROR - kdump.service stop failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 42'
+ message='../common/common_lib.sh line 42'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 42'
Fri Jan  6 13:27:15 2023 - ERROR - ../common/common_lib.sh line 42
+ return 0
+ systemctl start kdump.service
Failed to start kdump.service: Unit kdump.service not found.
+ CHECK_RESULT 5 0 0 'kdump.service start failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='kdump.service start failed'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'kdump.service start failed'
+ LOG_ERROR 'kdump.service start failed'
+ message='kdump.service start failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'kdump.service start failed'
Fri Jan  6 13:27:15 2023 - ERROR - kdump.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 44'
+ message='../common/common_lib.sh line 44'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 44'
Fri Jan  6 13:27:16 2023 - ERROR - ../common/common_lib.sh line 44
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen-riscv/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status kdump.service
+ grep 'Active: active'
Unit kdump.service could not be found.
+ CHECK_RESULT 1 0 0 'kdump.service start failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='kdump.service start failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'kdump.service start failed'
+ LOG_ERROR 'kdump.service start failed'
+ message='kdump.service start failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'kdump.service start failed'
Fri Jan  6 13:27:23 2023 - ERROR - kdump.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 47'
+ message='../common/common_lib.sh line 47'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 47'
Fri Jan  6 13:27:23 2023 - ERROR - ../common/common_lib.sh line 47
+ return 0
+ test_enabled kdump.service
+ service=kdump.service
++ systemctl is-enabled kdump.service
Failed to get unit file state for kdump.service: No such file or directory
+ state=
+ '[' '' == enabled ']'
+ '[' '' == disabled ']'
+ '[' '' == masked ']'
+ '[' '' == static ']'
+ LOG_INFO 'Unit is indirect, ignoring.'
+ message='Unit is indirect, ignoring.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Unit is indirect, ignoring.'
Fri Jan  6 13:27:24 2023 - INFO  - Unit is indirect, ignoring.
+ grep -i 'fail\|error'
+ journalctl --since '2023-01-06 13:26:57' -u kdump.service
+ grep -v -i 'DEBUG\|INFO\|WARNING'
+ CHECK_RESULT 1 0 1 'There is an error message for the log of kdump.service'
+ actual_result=1
+ expect_result=0
+ mode=1
+ error_log='There is an error message for the log of kdump.service'
+ '[' -z 1 ']'
+ '[' 1 -eq 0 ']'
+ test 1x == 0x
+ return 0
+ systemctl start kdump.service
Failed to start kdump.service: Unit kdump.service not found.
+ sed -i 's\kdumpctl start\kdumpctl restart\g' /usr/lib/systemd/system/kdump.service
sed: can't read /usr/lib/systemd/system/kdump.service: No such file or directory
+ systemctl daemon-reload
+ systemctl reload kdump.service
Failed to reload kdump.service: Unit kdump.service not found.
+ CHECK_RESULT 5 0 0 'kdump.service reload failed'
+ actual_result=5
+ expect_result=0
+ mode=0
+ error_log='kdump.service reload failed'
+ '[' -z 5 ']'
+ '[' 0 -eq 0 ']'
+ test 5x '!=' 0x
+ test -n 'kdump.service reload failed'
+ LOG_ERROR 'kdump.service reload failed'
+ message='kdump.service reload failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'kdump.service reload failed'
Fri Jan  6 13:27:29 2023 - ERROR - kdump.service reload failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_service_kdump.sh line 29'
+ message='oe_test_service_kdump.sh line 29'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_service_kdump.sh line 29'
Fri Jan  6 13:27:30 2023 - ERROR - oe_test_service_kdump.sh line 29
+ return 0
+ systemctl status kdump.service
+ grep 'active (exited)'
Unit kdump.service could not be found.
+ CHECK_RESULT 1 0 0 'kdump.service reload causes the service status to change'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='kdump.service reload causes the service status to change'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'kdump.service reload causes the service status to change'
+ LOG_ERROR 'kdump.service reload causes the service status to change'
+ message='kdump.service reload causes the service status to change'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'kdump.service reload causes the service status to change'
Fri Jan  6 13:27:31 2023 - ERROR - kdump.service reload causes the service status to change
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_service_kdump.sh line 31'
+ message='oe_test_service_kdump.sh line 31'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_service_kdump.sh line 31'
Fri Jan  6 13:27:31 2023 - ERROR - oe_test_service_kdump.sh line 31
+ return 0
+ LOG_INFO 'Finish test!'
+ message='Finish test!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish test!'
Fri Jan  6 13:27:32 2023 - INFO  - Finish test!
+ CASE_RESULT 0
+ case_re=0
+ test -z 8
+ test 8 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Fri Jan  6 13:27:32 2023 - ERROR - failed to execute the case.
+ exit 8
+ post_test
+ LOG_INFO 'start environment cleanup.'
+ message='start environment cleanup.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'start environment cleanup.'
Fri Jan  6 13:27:33 2023 - INFO  - start environment cleanup.
+ sed -i 's\kdumpctl restart\kdumpctl start\g' /usr/lib/systemd/system/kdump.service
sed: can't read /usr/lib/systemd/system/kdump.service: No such file or directory
+ systemctl daemon-reload
+ systemctl reload kdump.service
Failed to reload kdump.service: Unit kdump.service not found.
+ LOG_INFO 'Finish environment cleanup!'
+ message='Finish environment cleanup!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup!'
Fri Jan  6 13:27:38 2023 - INFO  - Finish environment cleanup!
